code/modules/research/rdconsole.dm

```diff
@@ -250,60 +250,63 @@ proc/CallMaterialName(ID)
 				linked_destroy.loaded_item.loc = linked_destroy.loc
 				linked_destroy.loaded_item = null
 				linked_destroy.icon_state = "d_analyzer"
-				screen = 2.1
+				screen = 1.0
 
 	else if(href_list["deconstruct"]) //Deconstruct the item in the destructive analyzer and update the research holder.
 		if(linked_destroy)
 			if(linked_destroy.busy)
 				usr << "<span class='danger'>The destructive analyzer is busy at the moment.</span>"
-			else
-				var/choice = input("Proceeding will destroy loaded item.") in list("Proceed", "Cancel")
+				return
+			var/list/temp_tech = linked_destroy.ConvertReqString2List(linked_destroy.loaded_item.origin_tech)
+			var/cancontinue = FALSE
+			for(var/T in temp_tech)
+				if(files.IsTechHigher(T, temp_tech[T]))
+					cancontinue = TRUE
+					break
+			if(!cancontinue)
+				var/choice = input("This item does not raise tech levels. Proceed destroying loaded item anyway?") in list("Proceed", "Cancel")

```

Comment:
This and the lines above make DA ask confirmation only when the origin tech levels aren't raising the R&D levels. Tested in different situations and it works.